src/Icon.js

```diff
@@ -22,7 +22,6 @@ export default class Icon extends Component {
 
     const newClassName = classNames(
       className,
-      `fi-${iconName}`,
```

Comment:
This should be changed to `iconName,` instead of being removed

Reply:
Done @bryanjos :smile: